proto/bpfman.proto

```diff
@@ -127,6 +133,7 @@ message KprobeAttachInfo {
     uint64 offset = 2;
     bool retprobe = 3;
```

Comment:
Do we still need retprobe in the attach info if it's also in the ProbeLoadInfo?

Reply:
Fixed